src/Native/Windows/build-native.cmd

```diff
@@ -111,6 +111,13 @@ echo See: https://github.com/dotnet/coreclr/blob/master/Documentation/project-do
 exit /b 1
 
 :GenVSSolution
+:: Generate Version file
+if not exist "%__IntermediatesDir%\clrcompression" md "%__IntermediatesDir%\clrcompression"
```

Comment:
Can we make this logic part of the msbuild target? I would like to keep the cmd/sh scripts as minimal as possible and do most of the processing inside of msbuild itself.